docs/application/native/guides/text-input/overview.md

```diff
@@ -1,13 +1,25 @@
 # Text Input and Voice
 
 
-The text input and voice features introduce how you can convert speech to text and back to speech, and provide customized keyboards and voice control.
+The text input and voice features introduce how you can convert speech to text and back to speech, and provide customized keyboards and voice control, and retrieve previous data in a text input field.
 
 You can use the following text input and voice features in your native applications:
 
 - [Input Method](input-method.md)
 
-  You can create an IME (Input Method Editor) application that provides a customized keyboard for the user. You can show and hide the keyboard as needed, and offer a keyboard option menu to allow the user to manage the keyboard settings.
+  You can create an Input Method Editor (IME) application that provides a customized keyboard for the user. You can show and hide the keyboard as needed, and offer a keyboard option menu to allow the user to manage the keyboard settings.
+
+- [Autofill](autofill.md)
+
+  You can get the retrieve the data stored in an autofill service.
```

Comment:
I think it is better to finalize the sentence before updating the sentence.
"You can get the retrieve the data stored in an autofill service."
This sentence is not correct. Therefore, can we rephrase as:
"You can retrieve the data stored in an autofill service."

I have a concern: The older sentence has been rephrased a lot. Please ensure that while doing the same we are not omitting any important information to a reader.
Need your opinion.

Reply:
How about the below sentence?
"You can get the data stored in the Autofill Service by the user previously entered."

Reply:
Are we trying to say:
"You can retrieve the previously entered data stored in an autofill service."

Reply:
Fixed.